reactivex/operators/_buffer.py

```diff
@@ -2,13 +2,15 @@
 
 from reactivex import Observable, compose
 from reactivex import operators as ops
+from reactivex.curry import curry_flip
+from reactivex.typing import UnaryOperator
 
 _T = TypeVar("_T")
 
 
 def buffer_(
     boundaries: Observable[Any],
-) -> Callable[[Observable[_T]], Observable[List[_T]]]:
+) -> UnaryOperator[_T, List[_T]]:
```

Comment:
UnaryOperator?

Reply:
Aha, now I see the definition below. Nice addition.

Reply:
Had to remove it because of a circular dependency. I'm not quite there with Python typing to be honest. Seems to confuse me, maybe cause I'm so used to TypeScript typing